api/turing/cluster/virtual_service.go

```diff
@@ -14,7 +14,7 @@ type VirtualService struct {
 	Endpoint        string            `json:"endpoint"`
 	DestinationHost string            `json:"destination_host"`
 	HostRewrite     string            `json:"host_rewrite"`
-	MatchURIPrefix  string            `json:"match_uri_prefix"`
+	MatchURIPrefix  []string          `json:"match_uri_prefix"`
```

Comment:
Maybe then also change the property field to `MatchURIPrefixes` to make it easy to read that multiple prefixes can be passed in